mlir/lib/Dialect/Transform/IR/TransformInterfaces.cpp

```diff
@@ -60,9 +60,13 @@ ArrayRef<Operation *>
 transform::TransformState::getPayloadOpsView(Value value) const {
   const TransformOpMapping &operationMapping = getMapping(value).direct;
   auto iter = operationMapping.find(value);
-  assert(
-      iter != operationMapping.end() &&
-      "cannot find mapping for payload handle (param/value handle provided?)");
+
+  if (iter == operationMapping.end()) {
+    value.dump();
```

Comment:
Leftover debug code

Reply:
Rewrite to `assert(iter != operationMapping.end() && "...")`

Reply:
I wanted to print the actual value before the assert, I think there si a convoluted Twine way but meh.